Sources/Styleguide/PointFreeBaseStyles.swift

```diff
@@ -464,7 +466,8 @@ private let aStyles =
   )
   <> (a & .pseudo(.hover) | a & .pseudo(.active)) % (
     key("text-decoration", "underline")
-)
+    )
+    <> (a & Class.pf.type.underlineLink) % key("text-decoration", "underline")
```

Comment:
this also shows up in the snapshot diffs everywhere. by default we don't underline links, so I need a way to explicitly bring back underlines.